src/containers/BookingPassengers/index.js

```diff
@@ -9,15 +9,22 @@ import {
 } from '../../components';
 import {colors} from '../../library/constants';
 
-export const SelectPassengers = ({navigation}) => {
-  const nextScreenCast = type =>
+export const SelectPassengers = ({route: {params}, navigation}) => {
+  
+  const nextScreenCast = () =>
     navigation.navigate('Received', {
```

Comment:
It is not recommended to upload the console.log() to GitHub, you can use some other option such as Alert. What do you think?